projects/storefrontlib/src/lib/product/components/product-details/product-summary/product-summary.component.html

```diff
@@ -1,24 +1,42 @@
-<div class="cx-product-summary">
-  <div class="cx-product-summary__price cx-product-summary__price" aria-label="new item price">
-    {{product?.price?.formattedValue}}
-  </div>
-
-  <div class="cx-product-summary__description">
-    <p [innerHTML]="product?.summary"> </p>
-  </div>
+<div class="product-summary">
+  <ng-container *cxOutlet="outlets.TITLE">
+    <div class="item-title">
+      <div class="name">{{ product?.name }}</div>
+      <div class="code">ID {{ product?.code }}</div>
+    </div>
+  </ng-container>
 
-  <div class="cx-product-summary__variant">
-    <div class="cx-product-summary__variant-label">
-      Qty:
+  <ng-container *cxOutlet="outlets.RATING">
+    <div class="item-rating">
+      <cx-star-rating [rating]="product?.averageRating" [disabled]="true"></cx-star-rating>
+      <div class="review-count">({{ product?.numberOfReviews }})</div>
+      <a class="review-link btn-link">Write a review</a>
     </div>
-    <cx-item-counter [min]="1" [max]="product.stock.stockLevel" *ngIf="product?.stock?.stockLevel > 0 || product?.stock?.stockLevelStatus === 'inStock'"
-      (update)="updateCount($event)"></cx-item-counter>
-    <span class="cx-product-summary__variant-info">
-      <span *ngIf="product?.stock?.stockLevel > 0 || product?.stock?.stockLevelStatus === 'inStock'">
-        {{product?.stock?.stockLevel}} in stock</span>
-      <span *ngIf="product?.stock?.stockLevel == 0">Out of stock</span>
-    </span>
-  </div>
+  </ng-container>
 
+  <div class="price" aria-label="new item price">
+    {{ product?.price?.formattedValue }}
+  </div>
+  <div class="description"><p [innerHTML]="product?.summary"></p></div>
 
-</div>
\ No newline at end of file
+  <ng-container *cxOutlet="outlets.ADDTOCART">
+    <div class="quantity">
+      <div class="quantity-label">Qty:</div>
```

Comment:
Can you have a conversation with Bill/PD team to see if we can get ride of this completely? Doesn't add any value from my pov. If we like to keep it, I'd say we make a word (quantity) and have it on 1 row with the quantity controls.

Reply:
I talked about it with Manu from PD and he gave me the following answer. The `qty` is to keep it consistent with the boxes we have in the checkout where writing `quantity` wouldn't fit. Also we are keeping it because when we have variants the design call for a similar label about the variants. 
![image](https://user-images.githubusercontent.com/10605724/48579331-2b39ce00-e8ea-11e8-946d-b5b1720b75db.png)

Reply:
But the styling changed and I will remove the colon.